docs/README.md

```diff
@@ -2,30 +2,22 @@
 sidebar_position: 1
 ---
 
-# Ignite - the battle-tested React Native boilerplate
+<p align="center"><img src="https://user-images.githubusercontent.com/1479215/206780298-2b98221d-9c57-4cd3-866a-cf85ec1ddd9e.jpg" alt="Ignite README Splash Image" /></p>
 
-With over seven years of active development, Ignite is the most popular React Native app starter boilerplate for both Expo and bare React Native.
+# Ignite - the battle-tested React Native boilerplate
 
-This is the React Native stack that the [Infinite Red](https://infinite.red) team uses on a day-to-day basis to build client apps. Developers who use Ignite report that it saves them two to four weeks of time on average off the beginning and improves their confidence over the lifetime of the entire project.
+<a href="https://badge.fury.io/js/ignite-cli" target="_blank"><img src="https://badge.fury.io/js/ignite-cli.svg" alt="npm version" height="20"></a>
+![GitHub Repo stars](https://img.shields.io/github/stars/infinitered/ignite)
+![Twitter Follow](https://img.shields.io/twitter/follow/ir_ignite)
+[![CircleCI](https://dl.circleci.com/status-badge/img/gh/infinitered/ignite/tree/master.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/infinitered/ignite/tree/master)
 
-## Getting Started
+## Proven React Native boilerplate
 
-If you just want to kick off a new project, you can do that with the Ignite CLI:
+Having been consistently developed and maintained since 2016, Ignite is the oldest active and most popular third-party React Native / Expo app boilerplate.
```

Comment:
```suggestion
Developed and maintained consistently since 2016, Ignite is the oldest active and most popular third-party React Native / Expo app boilerplate.
```

Reply:
Same as above.